src/components/filters/FilterRange.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {InputNumber} from "antd";
+import "antd/es/input-number/style/css";
+
+
+const FilterRange = ({
+                        filterTypeName,
+                        filterTypeMin,
+                        filterTypeMax,
+                        onChange,
+                        filters
+                      }) => {
+
+  const handleChange = (name, val) => {
+    onChange(name,val)
+  }
+
+  return <>
+    <div>
+      {`${filterTypeName.toUpperCase()}: `}
```

Comment:
`<label>{filterTypeName.toUpperCase()}</label>: ` would probably more appropriate.

When creating a form element label, you need to wrap it in a label and link it to your form element somehow. It makes it accessible to disabled people using assistive technologies. A website without a11y is like a building without a wheelchair ramp: not cool. 

Link: https://webaim.org/techniques/forms/controls
When you have some time read up on web-accessibility & ARIA